tidb-server/main.go

```diff
@@ -188,6 +191,36 @@ func syncLog() {
 	}
 }
 
+func initializeTempDir() {
+	tempDir := config.GetGlobalConfig().TempStoragePath
+	lockFile := "_dir.lock"
+	_, err := os.Stat(tempDir)
+	if err != nil && !os.IsExist(err) {
+		err = os.MkdirAll(tempDir, 0755)
+		terror.MustNil(err)
+	}
+	tempDirLock, err = fslock.Lock(filepath.Join(tempDir, lockFile))
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "acquire the lock of temp storage dir err:", err)
```

Comment:
Please use a more user friendly error, a good error message should describe:
1. What's the error(keep it short, use plain language, leave the techno-jargon out)?
2. What's the workaround?

Reply:
And it's better to add troubleshooting instructions [here](https://github.com/pingcap/docs/blob/master/faq/tidb.md#tidb-custom-error-messages)